frontend/src/features/admin-form/responses/FeedbackPage/FeedbackTable.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect } from 'react'
```

Comment:
nit: I don't think we need the `React` import, we should destructure the imports we need directly, like `useMemo`, `useCallback`, etc.